README.md

```diff
@@ -26,6 +26,14 @@ use this library are also provided:
 
 # Building
 
+## For macOS
```

Comment:
Can you please put this after the "For UNIX" section rather than before?

Then replace "below" => "above".